### PR TITLE
Add information about API Elements and subpackages to the README

### PR DESCRIPTION
**Pull request by kylef**
I'm sure we can improve on this more (#17), but I wanted to get something started in the README so we can start redirecting contributors from the existing Fury repositories over here as I've seen some PRs over in some of the adapters.

See [rendered version](https://github.com/apiaryio/api-elements.js/tree/kylef/readme).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,47 @@
-# API Elements
+# API Elements: JavaScript
 
-## Using Lerna
+[API Elements](https://apielements.org/) is a uniform interface for dealing
+with API Descriptions. This repository contains tooling for handling API
```
Comment (honzajavorek) [inline README.md]:
```suggestion
with API description formats ([API Blueprint](https://apiblueprint.org/), [OpenAPI](https://github.com/OAI/OpenAPI-Specification), ...). This repository contains tooling for handling API
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+Elements in JavaScript along with parsers and serializers for API Description
```
Comment (honzajavorek) [inline README.md]:
```suggestion
Elements in JavaScript along with parsers and serializers for API description
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,47 @@
...
+languages.
+
+API Element Adapters:
```
Comment (honzajavorek) [inline README.md]:
```suggestion
API Elements adapters:
```

File: README.md [hunk 4]
```diff
@@ -1,6 +1,47 @@
...
+- [OpenAPI 2 Parser](packages/fury-adapter-swagger)
+
+## Usage
+
+```javascript
+const { Fury } = require('fury');
+
+const fury = new Fury();
+
+// Load any parsers or serializer adapters you wise to use
```
Comment (honzajavorek) [inline README.md]: wise?

File: README.md [hunk 5]
```diff
@@ -1,6 +1,47 @@
...
+fury.use(openAPI2Parser);
+
+const source = 'FORMAT: 1A\n# My API');
+fury.parse({source}, (error, parseResult) => {
+  console.log(parseResult.api.title);
+});
+```
+
+See [API Reference documentation](https://api-elements-js.readthedocs.io/en/latest/api.html#elements)
+for the details about ParseResult object and other elements interface in
```
Comment (honzajavorek) [inline README.md]:
```suggestion
for the details about the ParseResult object and other elements interface in
```

File: README.md [hunk 6]
```diff
@@ -1,6 +1,47 @@
...
+const fury = new Fury();
+
+// Load any parsers or serializer adapters you wise to use
+const apiBlueprintParser = require('fury-adapter-apib-parser');
+fury.use(apiBlueprintParser);
+
+const openAPI2Parser = require('fury-adapter-swagger');
+fury.use(openAPI2Parser);
+
+const source = 'FORMAT: 1A\n# My API');
```
Comment (honzajavorek) [inline README.md]: Perhaps a multiline string which looks like _API description_ would do a better justice to the example:

```js
const source = `
FORMAT: 1A

# My API
## GET /message
+ Response 200 (text/plain)

        Hello World!
`;
```

(the document stolen from the APIB homepage)

File: README.md (hunk 2 above)
Comment (honzajavorek) [reply README.md]: Hmm, API description languages mean the same as API description formats. I wouldn't argue about domain dictionary now, but I suggested formats above and we should be at least consistent.

File: README.md (hunk 4 above)
Comment (kylef) [reply README.md]: wish*

**Comment by kylef** [comment]
Thanks @honzajavorek, I've taken all of your feedback in.

**Pull request merged**